client/src/modules/authorization/components/ResetPassword.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { PureComponent } from 'react';
+import validator from 'validator';
+import { FormattedMessage } from 'react-intl';
+
+class ResetPassword extends PureComponent {
+  state = {
+    email: '',
+    tipVisible: false
+  };
+
+  handleInputChange = event => {
+    const {
+      target: { value }
+    } = event;
+
+    this.setState({ email: value });
+  };
+
+  handleEmailValidation = () => {
+    const { email } = this.state;
+
+    return validator.isEmail(email);
+  };
+
+  handleSubmit = event => {
+    event.preventDefault();
+    const isEmailCorrect = this.handleEmailValidation();
+
+    if (isEmailCorrect) {
+      // handle submit here
+      return;
+    }
+
+    this.handleTipVisibility();
+  };
+
+  handleTipVisibility = () => {
+    this.setState({ tipVisible: true });
+    setTimeout(() => this.setState({ tipVisible: false }), 6000);
+  };
+
+  render() {
+    const { email, tipVisible } = this.state;
+
+    return (
+      <form className="reset-password" onSubmit={this.handleSubmit}>
+        <h2 className="reset-password__heading">
+          <FormattedMessage id="authorization.reset-password.heading" />
+        </h2>
+        <div className="reset-password__body">
+          <label className="reset-password__email-label">
+            <FormattedMessage id="authorization.reset-password.email-label" />
+            <input
+              className="reset-password__email-input primary-input"
+              onChange={this.handleInputChange}
+              type="email"
+              value={email}
+            />
+          </label>
+          {tipVisible && (
+            <span className="reset-password__message-error">
+              <FormattedMessage id="authorization.reset-password.tip-content" />
+            </span>
+          )}
+          <button
```

Comment:
Consider: disable button if the field is empty or its value is invalid.

Reply:
Only for an empty field.
When it filled even with a wrong value user should be able to press the button to get the TIP with the information about the mistake he made. Otherwise, the user will have no feedback, what he did wrong.